spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/src/main/java/org/springframework/cloud/gcp/core/autoconfig/GoogleCloudContextAutoConfiguration.java

```diff
@@ -14,26 +17,26 @@
 
 /**
  * @author Vinicius Carvalho
+ *
+ * Base starter for Google Cloud Projects. Provide defaults for {@link GoogleCredentials}.
+ * Binds properties from {@link org.springframework.cloud.gcp.core.GoogleCloudProperties}
+ *
  */
 @Configuration
 @ConditionalOnClass(GoogleCredentials.class)
-public class CredentialsAutoConfiguration implements EnvironmentAware {
+@EnableConfigurationProperties(GoogleCloudProperties.class)
+public class GoogleCloudContextAutoConfiguration {
```

Comment:
For what it's worth, I'd cast my vote for using GCP. GCP is as much of a brand as AWS.